src/test/java/com/epam/tam/unittests/CalculatorTest.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.tam.unittests;
+
+import com.epam.tat.module4.Calculator;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+	private Calculator calculator;
+	private static final double DELTA = 0.01;
+	
+	@BeforeClass
+	public void setUp(){
+		calculator = new Calculator();
+	}
+	
+	@Test(expectedExceptions = IllegalArgumentException.class, dependsOnGroups = {"basic"})
+	public void divideByZeroThrowsException() {
+		calculator.div(2, 0);
+	}
+	
+	@Test(dataProvider = "dataForSum", groups = {"basic"})
+	public void testSum(long first, long second, long expected) {
+		long result = calculator.sum(first, second);
+		Assert.assertEquals(result, expected);
+	}
+	
+	@DataProvider(name = "dataForSum")
```

Comment:
Class structure should be the following:
- imports
- fields declaration (which includes prepared Data such as DataProvider)
- @Before* @After* methods
- Tests methods
- private methods